ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -62,6 +62,21 @@ tests:
           cpu: "2"
           memory: 4Gi
     workflow: ipi-aws
+- as: e2e-console
+  run_if_changed: frontend/
```

Comment:
@yapei I am not sure if you need to add "optional: true" here. could you please try it?
cc @jianzhangbjz

Reply:
@kuiwang02  Is this https://steps.ci.openshift.org/ci-operator-reference#line211 the `optional` flag you are talking about?

Reply:
no. I mean it is for prow job, not step.
you could try it with adding it and without adding it.

Reply:
It should generate a `job` automatically, and you can set the `optional` flag in it.

Reply:
@yapei why do you remove optional: true?
@jianzhangbjz actually I do not ensure it 100%. do you think this test need optional: true?

Reply:
Comments left.